.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: publish
+
+on:
+  workflow_dispatch:
```

Comment:
Doesn't `workflow_dispatch` mean it's only manually triggered? https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch
```suggestion
```